### PR TITLE
ci: updated trivy dependency to v0.51.4

### PR DESCRIPTION
**Pull request by ChristofferNissen**
Replace trivy v0.45.1 with v.51.4 to resolve potential vulnerability with ECS.

Convert `report.Metadata.OS.Family` to string as it is a string under a custom struct type masking a string

Closes #397

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,12 +1,12 @@
 module github.com/project-copacetic/copacetic
 
-go 1.21
+go 1.22.0
```
Comment (ashnamehrotra) [inline go.mod:4]: could we specify this as just 1.22?

Comment (ChristofferNissen) [reply go.mod:4]: Sure, this is what the cli put, but can change it

Comment (ashnamehrotra) [reply go.mod:4]: it should be ok to run go mod tidy, is it resulting in an error?

Comment (ChristofferNissen) [reply go.mod:4]: It just puts the value back to 1.22.0 in go.mod

**Review by ashnamehrotra — APPROVED** [review]
LGTM, thank you!

**Pull request merged**